src/python/tests/core/datastore/corpus_tagging_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for corpus tagging helper functions."""
+
+from datastore import corpus_tagging
+from datastore import data_types
+from tests.test_libs import helpers
+from tests.test_libs import test_utils
+import unittest
+
+
+@test_utils.with_cloud_emulators('datastore')
+class CorpusTaggingTest(unittest.TestCase):
+  """Test corpus tagging helper functions."""
+
+  def setUp(self):
+    helpers.patch_environ(self)
+
+  def test_get_targets_with_correct_tag(self):
+    """Tests get_targets_with_tag returns the correct fuzz target."""
+    data_types.CorpusTag(tag='Test', fuzz_target='Test_fuzz_target').put()
```

Comment:
Nit: all lowercase, and it may be better to update tag to something like test_tag.

Reply:
Done